src/Microsoft.OData.Core/ODataVersionCache.cs

```diff
@@ -18,7 +18,12 @@ internal sealed class ODataVersionCache<T>
         /// <summary>
         /// Lazy constructing T for ODataVersion.V4.
         /// </summary>
-        private readonly SimpleLazy<T> v3;
+        private readonly SimpleLazy<T> v4;
+
+        /// <summary>
+        /// Lazy constructing T for ODataVersion.V4_01.

```

Comment:
>V4_01 [](start = 49, length = 5)

V401?

Reply:
Yes. It used to be ODataVersion.V4_01, but CA made me remove the underscore...

---
In reply to: [150167041](https://github.com/OData/odata.net/pull/989#discussion_r150167041) [](ancestors = 150167041)